src/core/Object3D.js

```diff
@@ -320,13 +320,23 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 			}
 
+			var position = this.parent ? this.getWorldPosition( worldPosition ) : this.position;
+
 			if ( this.isCamera ) {
 
-				m1.lookAt( this.position, vector, this.up );
+				m1.lookAt( position, vector, this.up );
 
 			} else {
 
-				m1.lookAt( vector, this.position, this.up );
+				m1.lookAt( vector, position, this.up );
+
+			}
+
+			if ( this.parent ) {
+
+				this.parent.updateMatrixWorld();
+				m2.getInverse( this.parent.matrixWorld );
+				m1.premultiply( m2 );
```

Comment:
Is it actually safe to perform this calculation? `m2` is not necessarily a pure rotation matrix. Using `Matrix4.extractRotation()` should make the code more robust...

Demo: https://jsfiddle.net/7gdux2ch/41/

Uncomment the usage of the method to see the difference.

Reply:
Added that in.

Reply:
Non-Uniform scaling is problematic in many aspects, see #3845.